src/Social/Messages/DataTransferObject/MessageInput.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kanvas\Social\Messages\DataTransferObject;
+
+use Spatie\LaravelData\Data;
+
+class MessageInput extends Data
+{
+    /**
+     * __construct
+     *
+     * @return void
+     */
+    public function __construct(
+        public ?int $parent_id = null,
+        public ?int $parent_unique_id = null,
```

Comment:
@FredPeal string and any string value needs to be at the end not at the beginig of the method